### PR TITLE
Managing user profile

### PR DESCRIPTION
**Pull request by mathieu-ducrot**
**Issue** https://github.com/smartbooster/sandbox/issues/3

## Description

Add the user profile form and action.

### CODE REVIEW

File: src/Security/Form/Type/UserProfileType.php [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+            ->setDefaults([
+                'translation_domain' => 'admin'
+            ])
+        ;
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function getName()
```
Comment (nicolas-bastien) [inline src/Security/Form/Type/UserProfileType.php]: not needed : https://github.com/symfony/symfony-docs/issues/5588

File: src/Resources/translations/admin.fr.xlf [hunk 2]
```diff
@@ -18,6 +18,18 @@
...
+                <source>form.label_password</source>
+                <target>Mot de passe</target>
+            </trans-unit>
+            <trans-unit id="form.label_password_confirmation">
+                <source>form.label_password_confirmation</source>
+                <target>Confirmation mot de passe</target>
+            </trans-unit>
+            <trans-unit id="fieldset.label_profile">
+                <source>fieldset.label_profile</source>
+                <target>My compte</target>
```
Comment (nicolas-bastien) [inline src/Resources/translations/admin.fr.xlf]: Mon compte

**Review by nicolas-bastien — CHANGES_REQUESTED** [review]

File: src/Security/Form/Type/UserProfileType.php (hunk 1 above)
Comment (mathieu-ducrot) [reply src/Security/Form/Type/UserProfileType.php]: i removed it on my last commit

File: src/Resources/translations/admin.fr.xlf (hunk 2 above)
Comment (mathieu-ducrot) [reply src/Resources/translations/admin.fr.xlf]: fixed it on the last commit

**Pull request merged**